nbs/Environments/12_EnvUncertainSocialDilemma.ipynb

```diff
@@ -32,21 +32,15 @@
     "from nbdev.showdoc import *"
```

Comment:
<p>Why did you change anything here at all?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BarfussLab/pyCRLD/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>